src/qtt/instrument_drivers/adapters/virtual_dac_instrument_adapter.py

```diff
@@ -1,24 +1,21 @@
-from typing import Dict
+from typing import Dict, Optional
 
-from qilib.configuration_helper import InstrumentAdapterFactory
-from qilib.configuration_helper.adapters import SpiModuleInstrumentAdapter
+from qilib.configuration_helper import InstrumentAdapterFactory, InstrumentAdapter
 from qilib.utils import PythonJsonStructure
-
-from qtt.instrument_drivers.gates import VirtualDAC
 from qtt.instrument_drivers.adapters.constants import CONFIG, BOUNDARIES, GATE_MAP, INSTRUMENTS, ADDRESS, \
     ADAPTER_CLASS_NAME
+from qtt.instrument_drivers.gates import VirtualDAC
 
 
-class VirtualDACInstrumentAdapter(SpiModuleInstrumentAdapter):
+class VirtualDACInstrumentAdapter(InstrumentAdapter):
     """ Adapter for the qtt VirtualDAC."""
 
-    def __init__(self, address: str) -> None:
-        super().__init__(address)
+    def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [invalid syntax](https://app.codacy.com/app/CumulonimbusCalvus/qtt/pullRequest?prid=4532354)

Reply:
Do you know why this is invalid syntax? Should I remove the typehints?